app/views/services/index.njk

```diff
@@ -72,7 +72,7 @@
 
     {% set allServiceTransactionsPath = routes.allServiceTransactions.index if has_live_account else routes.formattedPathFor(routes.allServiceTransactions.indexStatusFilter, 'test') %}
 
-    {% if has_live_account or env.ENABLE_TEST_REPORTS === 'true' %}
+    {% if has_live_account %}
```

Comment:
If I understand correctly I think this condition should be entirely removed?

The logic before read "If the flag is on, the new behaviour (always show)" OR "The existing behaviour (only show with live accounts). 

I think now it should read "The new behaviour (always show)"